content/body/skip-link.php

```diff
@@ -28,11 +28,11 @@
         "Please review our mobile friendly solution before you decide to choose this one.",
 ]); ?>
 
-<p>This is an variation of a traditional skip link seen on many websites today. They appear only when keyboard focus is applied to them. Note that while it works well on desktop, <strong>it fails on mobile, due to screen readers not passing
-    focus events to the mobile browser.</strong>  Screen reader users can focus into them, but partially sighted users may be initially confused when they cannot see what they are focusing into.</p>
+<p>This is a variation of a traditional skip link seen on many websites today. They appear only when keyboard focus is applied to them. Note that while it works well on desktop, <strong>it fails on mobile, due to screen readers not passing
+    focus events to the mobile browser.</strong>  Screen reader users can focus on them, but partially sighted users may be initially confused when they cannot see what they are focusing on.</p>
 
-<p>Unlike a lot of implementations, this example has, in my opinion, one really super helpful feature: we have two skip links pointing to each other. If a keyboard user triggers the skip link by accident, they can <strong>undo
-  their mistakes</strong> by pressing they ENTER key again, which will return focus back to where they came from.  <strong>This is useful for people who have hand tremors.</strong> This feature was thought up by my colleague Alison Hall during an accessibility hackathon.</p>
+<p>Unlike a lot of implementations, this example has, in my opinion, one super helpful feature: we have two skip links pointing to each other. If a keyboard user triggers the skip link by accident, they can <strong>undo
+  their mistakes</strong> by pressing the ENTER key again, which will return the focus to where they came from.  <strong>This is useful for people who have hand tremors.</strong> This feature was thought up by my colleague Alison Hall during an accessibility hackathon.</p>
```

Comment:
I think still including the "back" word is useful for clarity

```suggestion
  their mistakes</strong> by pressing the ENTER key again, which will return the focus back to where they came from.  <strong>This is useful for people who have hand tremors.</strong> This feature was thought up by my colleague Alison Hall during an accessibility hackathon.</p>
```